static/css/base.scss

```diff
@@ -356,6 +356,28 @@ $headings-font-family: "Google Sans", "Public Sans", $font-family-sans-serif;
   animation: spin 1s infinite linear;
 }
 
+#spinner-text {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#spinner-text::after {
+  content: "Still loading ...";
+  z-index: 10;
+  margin-top: 150px;
+  margin-left: 20px;
+  font-size: 20px;
+  color: white;
```

Comment:
is white text visible enough against the loading background? could you add a screenshot

Reply:
done